pkg/webhook/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -139,5 +139,5 @@ func (h *ValidatingHandler) InjectDecoder(d *admission.Decoder) error {
 // Register will regsiter application configuration validation to webhook
 func Register(mgr manager.Manager) {
 	server := mgr.GetWebhookServer()
-	server.Register("/validating-applicationconfigurations", &webhook.Admission{Handler: &ValidatingHandler{}})
+	server.Register("/validating-core-oam-dev-v1alpha2-applicationconfigurations", &webhook.Admission{Handler: &ValidatingHandler{}})
```

Comment:
I would vote for `/v1alpha2/validating-applicationconfigurations` if we want to add version in path.

Reply:
I think we need to come up with a uniformed way for webhooks register in vela. Let's add a PR after this